src/test/java/com/firebolt/jdbc/connection/FireboltConnectionServiceSecretTest.java

```diff
@@ -64,6 +54,11 @@ void notExistingDb() throws SQLException {
         assertEquals("Database my_db does not exist", assertThrows(FireboltException.class, () -> createConnection("jdbc:firebolt:?env=dev&account=dev", connectionProperties)).getMessage());
     }
 
+    @Test
+    void noAccount() {
+        assertEquals("Cannot connect: account is missing", assertThrows(FireboltException.class, () -> createConnection("jdbc:firebolt:db", connectionProperties)).getMessage());
+    }
+
```

Comment:
Just checking, this test is only V2, right?
Otherwise, in V1 account is optional.